### PR TITLE
[pt] Enable link checking and normalize links

### PR DESCRIPTION
**Pull request by chalin**
- Contributes to #3190
- Fixes broken links
- Normalizes file names, to match default locale (`en`)
- Normalizes links, e.g., add trailing slashes

/cc @emdneto @joaopgrassi @jpkrohling @edsoncelio - FYI, in case you'd be willing to review this PR
/cc @nate-double-u

### CODE REVIEW

**Review by edsoncelio — APPROVED** [review]
Thank you @chalin!!

**Comment by edsoncelio** [comment]
PTAL @brunoguidone @jessicalins @MrErlison

**Review by joaopgrassi — APPROVED** [review]

File: content/pt-br/devops.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Esta metodologia vai além da implementação de um conjunto de tecnologias, req
...
+Tradicionalmente, em organizações complexas com [aplicações monolíticas](/monolithic-apps/) de [alto acoplamento](/tightly-coupled-architecture/), o trabalho era geralmente fragmentado entre vários grupos, levando a um alto número de transferências de responsabilidade e longo prazo nas entregas. Cada vez que um componente ou atualização estava pronto, era disponibilizado em uma fila para o próximo time. Como cada pessoa trabalhava em apenas uma pequena parte do projeto, esta abordagem levava a uma falha de responsabilidade no processo como um todo. O objetivo dos times era transferir o trabalho para o próximo time, e não entregar a funcionalidade correta ao cliente - uma clara falta de alinhamento nas prioridades.
```
Comment (MrErlison) [inline content/pt-br/devops.md:13]: add '/pt-br/' in '/tightly-coupled-architecture/'

File: content/pt-br/distributed-apps.md [hunk 2]
```diff
@@ -5,20 +5,20 @@ category: conceito
...
 Em um ambiente nativo da nuvem, os componentes individuais normalmente são executados como [contêineres](/pt-br/container/) em um [cluster](/pt-br/cluster/).
 
-## Problema relacionado 
+## Problema relacionado
 
-A execução de uma aplicação em um único computador representa um único ponto de falha — se esse computador falhar, a aplicação ficará indisponível. 
-As aplicações distribuídas são frequentemente contrárias as aplicações [monolíticas](/monolithic-apps/). 
-Uma aplicação monolítica pode ser mais difícil de escalar, pois os vários componentes não podem ser redimensionados de forma independente. 
+A execução de uma aplicação em um único computador representa um único ponto de falha — se esse computador falhar, a aplicação ficará indisponível.
+As aplicações distribuídas são frequentemente contrárias as aplicações [monolíticas](/monolithic-apps/).
```
Comment (MrErlison) [inline content/pt-br/distributed-apps.md:16]: add '/pt-br/' in '/monolithic-apps/'

File: content/pt-br/microservices-architecture.md [hunk 3]
```diff
@@ -5,26 +5,26 @@ category: conceito
...
-Embora as aplicações modernas, como a Netflix, pareçam ser uma única aplicação, elas são na verdade uma coleção de serviços menores - todos trabalhando em colaboração. 
-Por exemplo, uma única página que permite acessar, pesquisar e visualizar vídeos provavelmente é alimentada por serviços menores que lidam com um aspecto (por exemplo, pesquisa, autenticação e execução de visualizações no seu navegador). 
+Os microsserviços têm uma abordagem moderna para o desenvolvimento de aplicações que aproveita as tecnologias nativas da nuvem.
+Embora as aplicações modernas, como a Netflix, pareçam ser uma única aplicação, elas são na verdade uma coleção de serviços menores - todos trabalhando em colaboração.
+Por exemplo, uma única página que permite acessar, pesquisar e visualizar vídeos provavelmente é alimentada por serviços menores que lidam com um aspecto (por exemplo, pesquisa, autenticação e execução de visualizações no seu navegador).
 Em resumo, os microsserviços referem-se a um padrão de arquitetura de aplicações geralmente contrária as [aplicações monolíticas](/monolithic-apps/).
```
Comment (MrErlison) [inline content/pt-br/microservices-architecture.md:11]: add '/pt-br/' in '/monolithic-apps/'

File: content/pt-br/tightly-coupled-architecture.md [hunk 4]
```diff
@@ -7,5 +7,4 @@ tags: ["fundamento", "arquitetura", "propriedade"]
...
 
-Arquiteturas acopladas de aplicativos são uma maneira bastante tradicional de criar aplicações. Embora não sejam necessariamente consistentes com todas as melhores práticas de desenvolvimento de [microsserviços](/microservices/), elas podem ser úteis em circunstâncias específicas. Elas tendem a ser mais rápidas e simples de implementar e, assim como [aplicações monolíticas](/monolithic-apps/), elas podem acelerar o ciclo de desenvolvimento inicial.
-
+Arquiteturas acopladas de aplicativos são uma maneira bastante tradicional de criar aplicações. Embora não sejam necessariamente consistentes com todas as melhores práticas de desenvolvimento de [microsserviços](/pt-br/microservices-architecture/), elas podem ser úteis em circunstâncias específicas. Elas tendem a ser mais rápidas e simples de implementar e, assim como [aplicações monolíticas](/monolithic-apps/), elas podem acelerar o ciclo de desenvolvimento inicial.
```
Comment (MrErlison) [inline content/pt-br/tightly-coupled-architecture.md:10]: add '/pt-br/' in '/monolithic-apps/'

**Comment by MrErlison** [comment]
Thank you @chalin
Just a few small details and after that, it will be LGTM!

**Review by emdneto — APPROVED** [review]

**Comment by chalin** [comment]
Anything left to do here @MrErlison et al.?

**Review by MrErlison — APPROVED** [review]

**Comment by chalin** [comment]
@nate-double-u - we seem to have the necessary approvals here. What else needs to be done?

**Comment by nate-double-u** [comment]
> @nate-double-u - we seem to have the necessary approvals here. What else needs to be done?

Nothing, merging!

**Pull request merged**